script/staging/FullStagingDeploy.s.sol

```diff
@@ -395,21 +399,21 @@ contract FullStagingDeploy is Script {
         bytes32 domainKeyBase     = RateLimitHelpers.makeDomainKey(controller.LIMIT_USDC_TO_DOMAIN(), CCTPForwarder.DOMAIN_ID_CIRCLE_BASE);
 
         // USDS mint/burn and cross-chain transfer rate limits
-        RateLimitHelpers.setRateLimitData(domainKeyBase,                   rateLimits, rateLimitData6,     "cctpToBaseDomainData",     6);
-        RateLimitHelpers.setRateLimitData(domainKeyArbitrum,               rateLimits, rateLimitData6,     "cctpToArbitrumDomainData", 6);
-        RateLimitHelpers.setRateLimitData(controller.LIMIT_USDC_TO_CCTP(), rateLimits, unlimitedRateLimit, "usdsToCctpData",           6);
-        RateLimitHelpers.setRateLimitData(controller.LIMIT_USDS_MINT(),    rateLimits, rateLimitData18,    "usdsMintData",             18);
-        RateLimitHelpers.setRateLimitData(controller.LIMIT_USDS_TO_USDC(), rateLimits, rateLimitData6,     "usdsToUsdcData",           6);
+        rateLimits.setRateLimitData(domainKeyBase,                   maxAmount6, slope6);
+        rateLimits.setRateLimitData(domainKeyArbitrum,               maxAmount6, slope6);
+        rateLimits.setRateLimitData(controller.LIMIT_USDC_TO_CCTP(), maxAmount6, slope6);
```

Comment:
`LIMIT_USDC_TO_CCTP` is an unlimitedRateLimit, whenever you see that, you do:

`rateLimits.setUnlimitedRateLimitData(key);`

Applies to all other `unlimitedRateLimits` you see in this PR (syrupWithdraw is one below). These also should be on their own line, away from the setRateLimitData groups.